contracts/ocr2/src/contract.rs

```diff
@@ -73,6 +73,9 @@ pub fn instantiate(
         billing: Billing {
             recommended_gas_price: 0,
             observation_payment: 0,
+            base_gas: None,
```

Comment:
Do we need `transmission_payment`?

Reply:
Good point, looks like that is missing altogether. I'll add it in 👍🏻

Reply:
Added